staff/albert-escolar/youtube-channel-template/index.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title>Paco's Channel</title>
+</head>
+
+<body>
+
+    <header>
+        <section>
+            <section class="searchbar">
+
+                <i class="fab fa-youtube"></i>
+                <h2>Paco's Channel</h2>
+
+                <button><i class="searchbar-sidebuttons fas fa-search"></i></button>
+                <button><i class="searchbar-sidebuttons fas fa-ellipsis-v"></i></button>
+
+            </section>
+    </header>
+
+    <div class="content">
+        <div class="side-navbar">
+            <div class="side-navbar-top-section">
+                <p>Inicio</p>
+                <p>Tendencias</p>
+                <p>Historial</p>
+            </div>
+
+            <div class="side-navbar-middle-section">
+                <h4>LO MEJOR DE YOUTUBE</h4>
+                <p>Música</p>
+                <p>Deportes</p>
+                <p>Videjuegos</p>
+                <p>Peliculas</p>
+                <p>Noticias</p>
+                <p>En Directo</p>
+                <p>Video en 360º</p>
+            </div>
+            <div class="side-navbar-explore">
+                <p>Explorar canales</p>
+            </div>
+            <div class="side-navbar-more">
+                <h4>MAS DE YOUTUBE</h4>
+                <p>Youtube Premium</p>
+                <p>Directo</p>
+            </div>
+            <div class="side-navbar-settings">
+                <p>Configuración</p>
```

Comment:
The same comment that [here](https://github.com/manuelbarzi/skylab-bootcamp-201904/pull/100/files#r272976927)